nav2_msgs/action/SmoothPath.action

```diff
@@ -1,5 +1,12 @@
-# Error codes
-# Note: The expected priority order of the errors should match the message order
+#goal definition
```

Comment:
All of the action definitions need proper capitalization + a space between the `#` and the comment for the changes. I'm not sure how those got that way with just copy pasting :wink:

Reply:
I added a space and capitalized the `error codes` and `note`. The definitions were always lower case. 

i.e `#goal definition`